### PR TITLE
Project of Property Tokenization by LABS

### PR DESCRIPTION
**Pull request by labsdev**
# Grant Application Checklist

- [ ] The [application-template.md](https://github.com/w3f/Open-Grants-Program/blob/master/applications/application-template.md) has been copied, renamed ( "project_name.md") and updated.
- [ ] A BTC address for the payment of the milestones is provided inside the application.  
- [ ] The software of the project will be released under the Apache license version 2.0 as specified in the [terms and conditions](https://gist.github.com/Noc2/75bc58e8ce9b5d419ff883b0cf2b8c19).
- [ ] The total funding amount of the project is below CHF 30k at the time of submission.

### CODE REVIEW

File: applications/labs_group.md [hunk 1]
```diff
@@ -0,0 +1,219 @@
+# Open Grant Proposal
```
Comment (labsdev) [inline applications/labs_group.md:1]:
```suggestion
# Open Grant Proposal
```

**Comment by labsdev** [comment]
This looks good

**Review by labsdev — COMMENTED** [review]
Grant-update

**Comment by labsdev** [comment]
signed

**Comment by mmagician** [comment]
This application is incomplete, I'm closing this for now. Next time please submit a filled-in application template as part of your first commit.

**Comment by labsdev** [comment]
Hello,

The link for submitting another application is not working. Do i have to
use the same previously submitted application and edit it instead?

Thanks in advance,

**Comment by mmagician** [comment]
@labsdev To submit an application, please follow [these steps](https://github.com/w3f/Open-Grants-Program#pencil-process).

**Pull request closed without merging**